packages/dds/merge-tree/src/test/client.spec.ts

```diff
@@ -31,6 +31,74 @@ describe("TestClient", () => {
 	});
 
 	describe(".findTile", () => {
+		it("Should not return tile when searching past the end of a string length 1", () => {
+			client.insertTextLocal(0, "");
```

Comment:
IIRC empty text segment inserts are just a no-op. so you should be able to remove this line. similar for next test case.